extensions/interactions/ItemSelectionInput/directives/item-selection-input-rules.service.ts

```diff
@@ -28,50 +26,31 @@ import { ItemSelectionRuleInputs } from 'interactions/rule-input-defs';
   providedIn: 'root'
 })
 export class ItemSelectionInputRulesService {
-  private removeDuplicatesInArrayPipe: RemoveDuplicatesInArrayPipe = (
-    new RemoveDuplicatesInArrayPipe());
-
   Equals(
       answer: ItemSelectionAnswer,
       inputs: ItemSelectionRuleInputs): boolean {
-    var normalizedAnswer = this.removeDuplicatesInArrayPipe.transform(answer);
-    var normalizedInput = this.removeDuplicatesInArrayPipe.transform(inputs.x);
-    return normalizedAnswer.length === normalizedInput.length &&
-        normalizedAnswer.every((val) => {
-          return normalizedInput.indexOf(val) !== -1;
-        });
+    return answer.length === inputs.x.length &&
+    answer.every(val => inputs.x.indexOf(val) !== -1);
```

Comment:
```suggestion
    answer.every(val => inputs.x.contains(val));
```
ditto in other places

Reply:
Done